core/util/src/main/scala/net/liftweb/util/HtmlParser.scala

```diff
@@ -238,11 +238,6 @@ trait Html5Writer {
               pc.buildString(sb)
               writer.append(sb)
             }
-            case pc: scala.xml.PCData => {
-              val sb = new StringBuilder()
-              pc.buildString(sb)
-              writer.append(sb)
-            }
```

Comment:
What happens if code hits this case after the removal of this code?

Reply:
One more I didn't comment, sorry. 
That branch was warned with a "core unreachable" warning. It's actually a duplicate of the pattern just above it, likely a merge that didn't end well.